src/test/java/org/opentripplanner/updater/trip/moduletests/addition/AddedTest.java

```diff
@@ -120,6 +127,65 @@ void repeatedlyAddedTripWithNewRoute() {
     assertNotNull(env.getTransitService().getRouteForId(firstRoute.getId()));
   }
 
+  @Test
+  public void addedTripWithSkippedStop() {
+    var env = RealtimeTestEnvironment.gtfs();
+    var builder = new TripUpdateBuilder(ADDED_TRIP_ID, SERVICE_DATE, ADDED, env.timeZone);
+    builder
+      .addStopTime(STOP_A1_ID, 30, DropOffPickupType.PHONE_AGENCY)
+      .addStopTime(STOP_B1_ID, 40, DropOffPickupType.COORDINATE_WITH_DRIVER, SKIPPED)
+      .addStopTime(STOP_C1_ID, 48, SKIPPED)
```

Comment:
Can you overload `addSkippedStop`?

Reply:
Sorry, not here. `addSkippedStop` requires a stop sequence only to skip stops for a scheduled trip, but here we need to specify the stop ID instead of the stop sequence.

Reply:
Add a new method with the same name.

Since I'm the reviewer, I get to decide what is appropriate.